src/main/cli/cli.c

```diff
@@ -611,11 +614,23 @@ static uint8_t getRateProfileIndexToUse()
     return rateProfileIndexToUse == CURRENT_PROFILE_INDEX ? getCurrentControlRateProfileIndex() : rateProfileIndexToUse;
 }
 
+#ifdef USE_OSD_PROFILES
+static uint8_t getOsdProfileIndexToUse()
+{
+    return osdProfileIndexToUse == CURRENT_PROFILE_INDEX ? getCurrentOsdProfileIndex() : osdProfileIndexToUse;
+}
+#endif
 
 static uint16_t getValueOffset(const clivalue_t *value)
 {
     switch (value->type & VALUE_SECTION_MASK) {
     case MASTER_VALUE:
+#ifdef USE_OSD_PROFILES
+        switch (value->type & VALUE_EXTENDED_SECTION_MASK) {
+            case OSD_PROFILE_VALUE:
+                return value->offset + sizeof(osdProfileName_t) * (getOsdProfileIndexToUse() - 1);
+        }
```

Comment:
Why the `switch` with only a single `case`?

Reply:
Not needed, thought it would be better for future expansion, but that may never happen.  Changed to if statement.